CONTRIBUTING.md

```diff
@@ -66,14 +63,25 @@ export CLIENT_ID={yourAppClientId}
 export USERNAME={userName}
 export PASSWORD={password}
 ```
-After setting up the environment variables, run this script to copy the configuration into the JSON configuration files:
 
-```bash
-sh scripts/setup-env.sh
-```
+For Windows, please set the following environment variables:
+- ISSUER
+- CLIENT_ID
+- USER_NAME
+- PASSWORD
+
+> **NOTE:** Windows has USERNAME as a built-in system variable, hence set the USER_NAME environment variable for testing.
 
 Then run the E2E tests:
 
 ```bash
 npm test
 ```
+
+**NOTE:** If you want to execute individual tests such as `npm run test:okta-hosted-login`, you will need to update the environment by running the following node script:
+
+```bash
+node scripts/setup-env.js
+```
+
+**NOTE:** If you want to test a different org or client app, you need to delete the configuration files `custom-login/src/app/.samples.config.ts` and `okta-hosted-login/src/app/.samples.config.ts`, and start from [first step](#running-e2e-tests-locally-before-commits)
```

Comment:
Ah, so now I see why you were manually copying them again in setup-env.js - it allows me to change the environment variables and then just run `npm test` and be happy.  I think I like that better.  @jmelberg-okta any prefernce?

Reply:
Agreed. We should allow the `.samples.config.js` files to be modifiable without explicitly requiring environment variables. It appears that with every `npm test` command executed we attempt to modify the files.